### PR TITLE
B 2207 Base project structure

### PR DESCRIPTION
**Pull request by Veronichka**
Configured base spring boot project structure. Configured cucumber and added primary acceptance tests feature file.

### CODE REVIEW

**Comment by clefevre** [comment]
Test comment

File: gradlew.bat [hunk 1]
```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (kurron) [inline gradlew.bat:1]: Do we really need to keep the Windows build file?

**Review by kurron — APPROVED** [review]
Looks good.  I'm assuming everything builds and runs?

**Comment by kurron** [comment]
I just pulled down the branch and built it.  I noticed it doesn't have a default task.  Did you clone the working template or did you edit things again?

File: gradlew.bat (hunk 1 above)
Comment (clefevre) [reply gradlew.bat:1]: I mean we get it for free.

**Review by clefevre — APPROVED** [review]

**Review by AnnaKhoma — APPROVED** [review]

**Comment by Veronichka** [comment]
@kurron yes. Everything builds and runs. 
Maybe I removed to much from working template. I'll add default task in the next PR.

**Comment by kurron** [comment]
Remember, this is a research project so you don't have to get everything peer reviewed.  Much of what you will discover is new to us as well.  I would advise you to keep things as simple as possible.  In fact, just using   http://start.spring.io/ to get you started is not a bad idea.  Most of what we will learn will come for the end result when you present your findings to us.

**Pull request merged**